tests/views/api_tests.py

```diff
@@ -38,10 +41,24 @@
 class ResourceAPITests(ArchesTestCase):
     graph_fixtures = ["Data_Type_Model"]
     data_type_graphid = "330802c5-95bd-11e8-b7ac-acde48001122"
+    non_legacy_resource_instanceid = "eb817333-2010-4cf5-a6e9-88003bfa8b64"
 
     @classmethod
     def setUpTestData(cls):
         super().setUpTestData()
+        add_users()
+
+        # add resource and tile not sourced from legacy_load_testing_package
```

Comment:
Could have used a resource from the `legacy_load_testing_package`, but I didn't want to add another test dependent on that package.